### PR TITLE
Remaining docs comments

### PR DESCRIPTION
**Pull request by twakes**
Here are the remainder of my Gravity Docs edits. I made some organizational changes to hopefully help readability as follows:

- Moved the configuration section of the Managing Clusters (clusters.md) to its own section. The clusters.md was already very long with this configuration section being a large portion of it. It seems like it deserved its own section and belonged after the install.md section. I'm open to the location in the navigation, though.
- Removed the remote management section and included some of that information in the Gravity Hub section. The remote management just referred to using Hub and was duplicative with other parts in the documentation.
- Added "Accessing Clusters" (access.md) section which is about accessing clusters without Hub. It is a small section that mostly refers readers to Teleport documentation and the Configuring access section in the new Configuring Clusters section. Open to including this in some other section of the docs but I felt like it needed its own section and could be expanded with more information.

### CODE REVIEW

File: docs/4.x/requirements.md [hunk 1]
```diff
@@ -104,7 +104,7 @@ These ports are used for Cluster operation and should be open between cluster no
 | 3022-3025               | SSH                                     | Teleport internal SSH control panel       |
 | 3080                    | HTTPS                                   | Teleport Web  UI                          |
 | 3008-3012               | HTTPS                                   | Internal Telekube services                |
-| 32009                   | HTTPS                                   | Telekube Cluster/OpsCenter Admin panel UI |
+| 32009                   | HTTPS                                   | Telekube Cluster/OpsCenter Control Panel UI |
```
Comment (benarent) [inline docs/4.x/requirements.md]: note: building with `make run-docs`, It's not showing older version / throwing a 404. We should fix this.

File: docs/6.x/hub.md [hunk 2]
```diff
@@ -3,20 +3,17 @@
 The Gravity Hub is a multi-cluster control plane available in the Enterprise
```
Comment (benarent) [inline docs/6.x/hub.md]: Would be cool to add screenshots to this, but that's my task :-)

File: docs/6.x/installation.md [hunk 3]
```diff
@@ -93,40 +93,40 @@ $ sudo ./gravity join 10.1.10.1 --advertise-addr=10.1.10.3 --token=XXX --role="w
...
 `--service-gid`      | _(Optional)_ Service group ID (numeric). See [Service User](pack/#service-user) for details. A group named `planet` is created automatically if unspecified.
-`--dns-zone`         | _(Optional)_ Specify an upstream server for the given DNS zone within the cluster. Accepts `<zone>/<nameserver>` format where `<nameserver>` can be either `<ip>` or `<ip>:<port>`. Can be specified multiple times.
+`--dns-zone`         | _(Optional)_ Specify an upstream server for the given DNS zone within the Cluster. Accepts `<zone>/<nameserver>` format where `<nameserver>` can be either `<ip>` or `<ip>:<port>`. Can be specified multiple times.
 `--vxlan-port`       | _(Optional)_ Specify custom overlay network port. Default is `8472`.
-`--exclude-from-cluster` | _(Optional)_ Excludes this node from the cluster, i.e. allows to bootstrap the cluster from a developer's laptop, for example. In this case the Kubernetes master will be chosen randomly.
+`--exclude-from-Cluster` | _(Optional)_ Excludes this node from the Cluster, i.e. allows to bootstrap the Cluster from a developer's laptop, for example. In this case the Kubernetes master will be chosen randomly.
```
Comment (a-palchikov) [inline docs/6.x/installation.md]: `--exclude-from-cluster` has been renamed to `--remote`

File: docs/6.x/overview.md [hunk 4]
```diff
@@ -87,9 +89,10 @@ Gravity consists of the following components:
 |-------------|---------------------------------------------------------------------------------------------------------------------------------------------------------|
 | `tele`      | The build tool. `tele` is used for building Cluster Images. The enterprise edition of `tele` can also publish Cluster Images into Gravity Hub.  |
 | `tsh`       | The remote access client to authenticate against a SAML/OAuth identity provider and remotely connect to Kubernetes clusters via SSH or Kubernetes API.   |
-| `gravity` <small>service</small> | The Cluster manager agent which runs on every  node. It reduces the complexities of Kubernetes management.
-| `gravity` <small>CLI tool</small>  | The CLI tool to perform high level Cluster administration tasks like expansion, upgrades, etc.  |
-| Gravity Hub | Gravity Hub is a web portal and repository for publishing Cluster Images. Think of Gravity Hub as a catalog of Kubernetes clusters and Kubernetes applications. Gravity Hub is only available in the Gravity Enterprise edition. |
+| `gravity`(agent) | The Cluster manager agent which runs on every  node. It reduces the complexities of Kubernetes management.
```
Comment (a-palchikov) [inline docs/6.x/overview.md]: Not sure agent needs to be mentioned - it is part of the functionality of the gravity binary and it more of an implementation detail. What I'd mention is the planet container.

**Comment by benarent** [comment]
Since I added my screenshots to these images, I fixed the smaller issues along the way.

File: docs/6.x/installation.md [hunk 5]
```diff
@@ -1,57 +1,56 @@
...
+  Linux hosts are compatible with Gravity.
+* The hosts should be clean without Docker or any container
+  orchestrator running on them.
+* The hosts should be able to connect to each other, i.e. on the same
   private network.
-* You should have the ability to create DNS entries for public access points.
-* You you should be able to obtain a valid SSL/TLS certificates for HTTPS.
-
+* You need the ability to create DNS entries for public access points.
+* You need to be able to obtain a valid SSL/TLS certificates for HTTPS.
```
Comment (webvictim) [inline docs/6.x/installation.md]: Should be either `obtain valid SSL/TLS certificates` or `obtain a valid SSL/TLS certificate`

File: docs/6.x/installation.md [hunk 6]
```diff
@@ -93,40 +93,40 @@ $ sudo ./gravity join 10.1.10.1 --advertise-addr=10.1.10.3 --token=XXX --role="w
...
 the most frequently used ones are listed below:
 
 Flag               | Description
 -------------------|-------------
-`--token`          | Secure token which prevents rogue nodes from joining the cluster during installation. Carefully pick a hard-to-guess value.
+`--token`          | Secure token which prevents rogue nodes from joining the Cluster during installation. Carefully pick a hard-to-guess value.
 `--advertise-addr` | The IP address this node should be visible as. **This setting is mandatory** to correctly configure Kubernetes on every node.
 `--role`           | _(Optional)_ Application role of the node.
-`--cluster`        | _(Optional)_ Name of the cluster. Auto-generated if not set.
+`--Cluster`        | _(Optional)_ Name of the Cluster. Auto-generated if not set.
```
Comment (webvictim) [inline docs/6.x/installation.md]: This should be `--cluster` as it's an argument to the CLI tool

File: docs/6.x/overview.md (hunk 4 above)
Comment (webvictim) [inline docs/6.x/overview.md]: `every node` (remove double space)

Comment (twakes) [reply docs/6.x/overview.md]: @a-palchikov could you suggest some language? As an aside, I noticed in general we don't talk much about Planet.

File: docs/6.x/config.md [hunk 7]
```diff
@@ -0,0 +1,854 @@
...
+Resource Name             | Resource Description
+--------------------------|---------------------
+`github`                  | GitHub connector
+`oidc`                    | OIDC connector (Enterprise only)
+`saml`                    | SAML connector (Enterprise only)
+`role`                    | Cluster role
+`user`                    | Cluster user
+`token`                   | user tokens such as API keys
+`logforwarder`            | forwarding logs to a remote rsyslog server
+`trusted_Cluster`         | managing access to remote Gravity Hubs (Enterprise Only)
```
[the remark this answers is not in the document]
Comment (twakes) [reply docs/6.x/config.md]: @a-palchikov Could you push your suggested changes directly to this branch?

**Pull request merged**